generators/app/templates/app/middlewares/docsAuth.ejs

```diff
@@ -0,0 +1,19 @@
+const { decode } = require('jwt-simple');
+
+const logger = require('../logger');
+
+const isDocumentationAuthenticated = async (req, res, next) => {
+  const token = req.headers['x-access-token'];
+
+  if (!token) return next({ internalCode: 401, message: 'The token was not given' });
+
+  const decodedToken = await decode(token, process.env.DOCUMENTATION_TOKEN_SECRET);
```

Comment:
why are you using jwt? instead of simply comparing the header with the token you have as secret. 
we dont need to encrypt anything